### ISSUE TITLE
about HO3D.py

### ISSUE THREAD

**Issue by rookiiiie** [namepllet/HandOccNet#30]
hello！Thanks for your work.

Can you tell me how to gain the file 'HO3Dv3_partial_test_multiseq_coco.json'  in HO3D.py?

`db = COCO(osp.join(self.annot_path, 'HO3Dv3_partial_test_multiseq_coco.json'))`

**Comment by yxt7979** [comment]
Hiii, I've been reproducing this project recently, and I found that there are some lines with "# TEMP", after I comment these and change these lines to the old ones, It works.

However, I found that the result is far cry from that in paper: https://github.com/namepllet/HandOccNet/issues/19

What about your results? Could you reproduce this successfully?

**Comment by rookiiiie** [comment]
no...i think the file "HO3D_train_data.json" miss necessary information:(

**Comment by yxt7979** [comment]
![image](https://github.com/namepllet/HandOccNet/assets/62125435/3fd15b9c-01f2-4bae-bd6e-332f2a095544)
Hii, after I get to the previous code version, it works..... 🤕

**Comment by M-leng** [comment]
@yxt7979 Could you share the previous code?
